src/nginxconfig/generators/conf/wordpress.conf.js

```diff
@@ -60,6 +60,7 @@ export default global => {
         config['location = /wp-login.php'] = {
             limit_req: 'zone=login burst=2 nodelay',
             include: 'nginxconfig.io/php_fastcgi.conf',
+            fastcgi_pass: 'unix:/run/php/php7.4-fpm.sock',
```

Comment:
This does not respect the user's selection of PHP server -- this will either need to become a file that is generated per domain so that you can consume the correct value here, or you could consider moving this block of logic to the main domain generator file instead?

Reply:
which one is main domain generator file?

Reply:
https://github.com/digitalocean/nginxconfig.io/blob/master/src/nginxconfig/generators/conf/website.conf.js :)

Reply:
Pushed a commit as suggested accordingly let me know if it looks fine

Reply:
@MattIPv4 tested it successfully as well.